### ISSUE TITLE
embed from unsupported platforms

### ISSUE THREAD

**Issue by agitator** [plone/plone.app.mosaic#144]
Some platforms are not supported within the embed tile or not worth to be added for a single case.
Is there a tile that allows to add some sort of html-snippets?

**Comment by datakurre** [comment]
Not really. Not sure, if we want to enable such tile by default. You could try theme fragment tile provided by collective.themefragments. They can have configuration by adding configuration schema as supermodel XML fragments/[filename].xml, but that's quite experimental and not really documented.

**Comment by tomgross** [comment]
Noembed plugins are ease to implement but the project seems unmaintained 
https://github.com/leedo/noembed/pull/54

**Comment by vangheem** [comment]
I think we should get rid of noembed and just have an html embed tile where you paste the html into a field and it renders.

**Comment by datakurre** [comment]
@vangheem I agree that we should support RAW html embed/injections, but how much should we protect uses from themselves? Validating the code through lxml? Requiring custom permission?

Yet, also oembed should be supported (if the embed value looks like an url). It's nice convention. Noembed was used as a quick and dirty solution, because there was not yet "perfect" oembed library for Python.
